packages/agent/src/agent/http/index.ts

```diff
@@ -392,6 +420,14 @@ export class HttpAgent implements Agent {
     return this.rootKey;
   }
 
+  public invalidateIdentity(): void {
```

Comment:
Is this used only for testing purposes? or is this part of the API to be used somewhere else?

Reply:
IIUC this would be used later to trigger a 'logout' if a session expires (implementing an idle timeout), right?

Reply:
Yes - this is the lower level API to support an inactivity logoout